ui/src/app/components/sharded-bulk-source-details-form/sharded-bulk-source-details-form.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ShardedBulkSourceDetailsFormComponent } from './sharded-bulk-source-details-form.component';
+
+describe('ShardedBulkSourceDetailsFormComponent', () => {
+  let component: ShardedBulkSourceDetailsFormComponent;
+  let fixture: ComponentFixture<ShardedBulkSourceDetailsFormComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ ShardedBulkSourceDetailsFormComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ShardedBulkSourceDetailsFormComponent);
```

Comment:
It is not clear to me where TestBed is being used

Reply:
This is Angular auto-generated code which gets added when a new component is created. 
Ref - https://github.com/search?q=repo%3Acloudspannerecosystem%2Fharbourbridge+TestBed.createComponent&type=code